e2e/internal/upgrade/upgrade_test.go

```diff
@@ -101,7 +101,7 @@ func TestUpgrade(t *testing.T) {
 	log.Println(string(data))
 
 	log.Println("Triggering upgrade.")
-	cmd := exec.CommandContext(context.Background(), cli, "upgrade", "apply", "--force", "--debug")
+	cmd := exec.CommandContext(context.Background(), cli, "upgrade", "apply", "--force", "--debug", "--yes", "--tf-log=DEBUG")
```

Comment:
We need to check if the binary supports `--tf-log` before adding it to the command.

Reply:
How would you do that? Run it with the `--help` flag and see if the output contains `tf-log`?

Reply:
Yes. One example can be found here: https://github.com/edgelesssys/constellation/blob/main/.github/actions/constellation_create/action.yml#L211-L215

Dunno if there are better ways, but this seems like a pretty accurate check to me.